webrtc.html

```diff
@@ -4488,11 +4495,6 @@
                 <p>The <code>errorCode</code> attribute is the numeric STUN
                 error code returned by the STUN or TURN server
                 [[STUN-PARAMETERS]].</p>
-                <p>If no host candidate can reach the server,
```

Comment:
I don't think this text should be removed. It's normal to expect an attribute's description to mention all its possible values, without having to look elsewhere in the spec.